packages/cactus-test-verifier-client/src/main/typescript/verifier-with-go-eth-stress-check.ts

```diff
@@ -84,84 +96,51 @@ async function setupEnvironment(): Promise<void> {
   log.info("Prune Docker...");
   await pruneDockerAllIfGithubAction({ logLevel: testLogLevel });
 
-  log.info(`Start Ledger ${imageName}:${imageVersion}...`);
-  ledger = new OpenEthereumTestLedger({
-    imageName,
-    imageVersion,
-    emitContainerLogs: true,
-    logLevel: sutLogLevel,
+  log.info(`Start Ledger ${containerImageName}:${containerImageVersion}...`);
+  ledger = new GethTestLedger({
+    containerImageName,
+    containerImageVersion,
   });
   await ledger.start();
   const ledgerRpcUrl = await ledger.getRpcApiWebSocketHost();
   log.info(`Ledger started, RPC: ${ledgerRpcUrl}`);
 
-  // Create Test Account
-  constTestAcc = await ledger.createEthTestAccount(constTestAccBalance);
-
-  // Generate connector private key and certificate
-  const pkiGenerator = new SelfSignedPkiGenerator();
-  const pki = pkiGenerator.create("localhost");
-  connectorPrivKeyValue = pki.privateKeyPem;
-  connectorCertValue = pki.certificatePem;
-  const jwtAlgo = "RS512";
-
-  const connectorConfig: any = {
-    sslParam: {
-      port: 0, // random port
-      keyValue: connectorPrivKeyValue,
-      certValue: connectorCertValue,
-      jwtAlgo: jwtAlgo,
-    },
-    logLevel: sutLogLevel,
-    ledgerUrl: ledgerRpcUrl,
+  const listenOptions: IListenOptions = {
+    hostname: "0.0.0.0",
```

Comment:
@outSH It's been reported earlier by some contributors that if you try to bind to `0.0.0.0` in cloud instances (that are used as dev machines through VSCode's Remote SSH Dev extension) then it will fail due to some security policy on Google Cloud Platform (might be other cloud providers as well but the specific one I remember for sure was GCP)
Is there any chance that you could bind to `127.0.0.1` instead and that way we support the development flow described above as well? If there's a specific reason why it has to be bound to `0.0.0.0` then I'm OK with that too, I just figured I'd at least ask.

Reply:
@petermetz Thanks for information! I've searched for other uses of `0.0.0.0` in tests and scripts and fixed them as well (I've run the tests for majority of them)

Reply:
@outSH Awesome, thank you very much!